use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -285,7 +334,7 @@ main(int argc, char *argv[])
             break;                      /* ... so terminate loop */
         }
 
-        switch (req.mtype) {
+        switch ((int) req.mtype) {
```

Comment:
How come this is necessary?

Reply:
This is a `.c` file which now needs to be compiled with C++ compiler as we are injecting spdlog calls into it. Spdlog is a C++ library.

I think clang on Mac/OSX complains if this `int` typecasting is not there. I forgot the exact compiler error.